src/facebook_example/bot.clj

```diff
@@ -51,18 +63,30 @@
     (reaction/thank-for-attachment)))
 
 ; You should not need to touch the following code :)
-(defn postback? [messaging-event](contains? messaging-event :postback))
-(defn attachments? [messaging-event] (contains? (:message messaging-event) :attachments))
-(defn message? [messaging-event] (contains? messaging-event :message))
+(defn process-event [event]
+  ; The order of the matchers is important!
+  (match [event]
+    ; The user `sender-id` has selected one quick-reply option
```

Comment:
We're no longer matching on `sender-id`, yet we still mention it here. IMO can just be 

> The user has selected an quick-reply option.